File: 0009-palindrome-number/0009-palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        if (x<0) return false;
        else{
            string x_ = to_string(x);
            int i = 0;
            int j = x_.size()-1;
            while(i<=j){
                if (x_[i]==x_[j]){
                    i++; j--;
                }
                else return false;
            }
            return true;
        }
    }
};

File: 0009-palindrome-number/0009-palindrome-number.py
class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        if x<0: return False
        x_=str(x)
        for i in range(len(x_)/2):
            if x_[i] != x_[len(x_)-1-i]:
                return False
        return True
        